scripts/generate-version-info.sh

```diff
@@ -13,11 +13,15 @@ fi
 OUTPUT_FILE="$1"
 
 # packages
-sudo rpm --query --all --queryformat '\{"%{NAME}": "%{VERSION}-%{RELEASE}"\}\n' | jq --slurp --sort-keys 'add | {packages:(.)}' > "$OUTPUT_FILE"
+sudo rpm --query --all --queryformat '\{"%{NAME}": "%{VERSION}-%{RELEASE}"\}\n' | jq --slurp --sort-keys 'add | {packages:(.)}' > "${OUTPUT_FILE}"
+
+# version detection
+KUBELET_VERSION=$(kubelet --version | cut -d' ' -f2)
+AWSCLI_VERSION=$(aws --version | cut -d' ' -f1 | cut -d'/' -f2)
```

Comment:
nit: move these variables immediately prior to their usage under the `# binaries` comment.